packages/nimble-components/src/table-column/base/template.ts

```diff
@@ -1,10 +1,22 @@
-import { html } from '@microsoft/fast-element';
+import { html, ref, slotted } from '@microsoft/fast-element';
 import type { TableColumn } from '.';
 
+// prettier-ignore
 export const template = html<TableColumn>`
     <template slot="${x => x.columnInternals.uniqueId}">
-        <span class="header-content">
-            <slot></slot>
+        <span
+            ${ref('headerSpan')}
+            class="header-content"
+            @mouseover="${x => {
+        x.isValidContentAndHasOverflow = !!x.slottedHeaderContent.map(element => element.textContent?.trim()).join(' ')
```

Comment:
Can you move the duplicated logic for formatting the text to a method

Reply:
Done